### PR TITLE
Final React Redux Capstone 

### PR DESCRIPTION
**Pull request by codecaiine**
- Configured React-Redux Linters.
 - Gitflow was used for collaboration.
 - Documented work in a professional way.
 - Redux: Fetch data and update Redux store.
 - Render UI - lists. useSelector was used for rendering.
 - Redux: Write actions and reducers for booking rockets and joining missions.
 - Redux: Write actions and reducers for canceling rockets and leaving missions.
 - Render UI: conditional components rendering.
 - Render UI: My Profile section.
 - Tasks inside the additional requirements were also incorporated for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+
+![screenshot](./src/assets/img/capstone1.PNG)
+
+![screenshot](./src/images/img/capstone2.PNG)
+
+![screenshot](./src/images/img/capstone3.PNG)
+
+![screenshot](./src/images/capstone4.PNG)
+
+![screenshot](./src/images/capstone5.PNG)
```
Comment (helman101) [inline README.md]: - [x] These 3 screenshots are not working, kindly fix this issue 😄

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Hi @codecaiine   👋, 

Nice work so far on this milestone of the project  😄, but there are still some changes to do.

### To Highlight 🌟 
- You have done an incredible job with this project, keep it that way! 😄 
- You did a nice job with the styling 🎨 

### Required Changes ♻️

- [ ] I don't see any unit test in your project, kindly add unit test for each component😄
    - Remember that you have to use `jest` and `React Testing Library`.

- [ ] _Check the comments below_

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wandji20 — APPROVED** [review]
## Hi @codecaiine & @Isabellanamutenda  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 

Your project has been approved, however, Keep up the good work.

## Congratulations :+1:  :fireworks: 

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Hi @wandji20 ,
Thank you for reviewing our project and your approval.
Cheers !

**Pull request merged**